prow/plugins/lgtm/lgtm.go

```diff
@@ -60,9 +60,36 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	configInfo := map[string]string{}
+	for _, orgRepo := range enabledRepos {
+		parts := strings.Split(orgRepo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", orgRepo)
+		}
+		opts := optionsForRepo(config, parts[0], parts[1])
+		var isConfigured bool
+		var configInfoStrings []string
+		configInfoStrings = append(configInfoStrings, "The plugin has the following configuration:<ul>")
+		if opts.ReviewActsAsLgtm {
+			configInfoStrings = append(configInfoStrings, `<li>Reviews of "approve" or "request changes" act as adding or removing the lgtm label.</li>`)
+			isConfigured = true
+		}
+		if opts.StoreTreeHash {
+			configInfoStrings = append(configInfoStrings, "<li>Git tree_hash is stored inside a comment to detect squashed commits before removing lgtm labels.</li>")
```

Comment:
Isn't this overly technical?

I think it would be nice if this was something like:

* Pushing commits to the PR always removes LGTM (!StoreTree && !StickyLGTM)

or else:

* Pushing commits to the PR always removes LGTM, unless
  - Squashing commits does not remove LGTM
  - Commits from TEAM do not remove LGTM

Not sure we need to explain the technical details of how we get these features to work (tracking LGTMd tree hash)

Reply:
I have simplified the texts in `configInfo`, thanks for the very helpful suggestions :)